tools/sdk/uxdProtocol/uxdClient.ts

```diff
@@ -139,10 +139,9 @@ export const getControllerPda = (uxdProgramId: PublicKey): PublicKey => {
   return findAddrSync([Buffer.from('CONTROLLER')], uxdProgramId)[0]
 }
 
-// We do not need the mint symbol so it is just set with a placeholder value
 export const instantiateController = (
   uxdProgramId: PublicKey,
-  mintDecimals: number,
+  mintDecimals = 6,
```

Comment:
Ok as long as we're very careful with these defaults

Reply:
Missed this one. It's in UXDClient as UXD_DECIMALS.

https://github.com/UXDProtocol/uxd-client/blob/9e521a7da5a80c9f445c3ea0d9b5c81b927ccfce/src/utils.ts#L19

There should be no hardcoded value. If it's not in the UXD_CLIENT then it should be added.

Reply:
Also bellow I see depository decimal to 6? Why. Why put default values that make no sense, it's easy to overlook later.

Rather have it onsite where it's use with either the proper value or "value not used, passing DEFAULT_DECIMALS or something. Tiny things but put all together it become quite hard to reason about and overlook issues.

<img width="486" alt="Screenshot 2022-01-24 at 10 39 12" src="https://user-images.githubusercontent.com/40299747/150758229-4a4abd98-0192-4ded-85d4-b0d0fbabe69f.png">

Reply:
Would rather get rid of the `export const instantiateMangoDepository = (` and use the original one. Gain is so small, and possible obfuscation/confusion later on makes it not worth imo

Reply:
Same for `export const instantiateController = (`